includes/dashboard/showSelectedGroup.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/* 
+ * this is to show the selected group of the users, selected by the admin
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+
+if ( ! isset($_POST['groupId']) )
+{
+	echo "Sorry, there is some problem";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->